lib/devise/passkeys/controllers/concerns/reauthentication_challenge.rb

```diff
@@ -7,12 +7,12 @@ module Concerns
         module ReauthenticationChallenge
           extend ActiveSupport::Concern
 
-          def passkey_reauthentication_challenge_session_key
+          def Reauthentication_challenge_session_key
             "#{resource_name}_current_reauthentication_challenge"
           end
 
           def store_reauthentication_challenge_in_session(options_for_authentication:)
-            session[passkey_reauthentication_challenge_session_key] = options_for_authentication.challenge
+            session[Reauthentication_challenge_session_key] = options_for_authentication.challenge
```

Comment:
This needs to match the casing of the original, since it's the method that's called to get the key for the hash